$NetBSD: patch-plugins_ZamVerb_Makefile,v 1.1 2022/04/06 16:32:21 nia Exp $

- Do not do static-linking on non-Linux.
- The lv2 plugin segfaults during generation, disable it.

--- plugins/ZamVerb/Makefile.orig	2020-12-20 06:54:49.000000000 +0000
+++ plugins/ZamVerb/Makefile
@@ -41,13 +41,8 @@ endif
 # --------------------------------------------------------------
 # Extra flags
 
-ifeq ($(LINUX),true)
 BASE_FLAGS += $(shell pkg-config --cflags samplerate fftw3f)
 LINK_FLAGS += $(shell pkg-config --libs fftw3f samplerate)
-else
-BASE_FLAGS += $(shell pkg-config --static --cflags samplerate fftw3f)
-LINK_FLAGS += $(shell pkg-config --static --libs fftw3f samplerate)
-endif
 
 LINK_FLAGS += -lpthread
 
@@ -57,12 +52,6 @@ ifeq ($(HAVE_JACK),true)
 TARGETS += jack
 endif
 
-ifeq ($(HAVE_DGL),true)
-TARGETS += lv2_sep
-else
-TARGETS += lv2_dsp
-endif
-
 TARGETS += vst
 
 all: $(TARGETS)
